modules/standard/Random.chpl

```diff
@@ -87,7 +87,56 @@ module Random {
   }
 
   /* The default RNG. The current default is PCG - see :mod:`PCGRandom`. */
-  param defaultRNG = RNG.PCG;
+  config param defaultRNG = RNG.PCG;
+
+  private
+  var globalRandomStream:
+    if defaultRNG == RNG.PCG then PCGRandom.RandomStream(eltType=real, parSafe=true)
+    else NPBRandom.NPBRandomStream(eltType=real, parSafe=true);
+
+  {
+    var seedVal: uint(64) = 0;
+    try {
+      var f = open("/dev/urandom", iomode.r);
```

Comment:
Not all platforms will have "/dev/urandom". So I'm not sure if it's reasonable to make any program using Random not run in such an environment. I'd consider printing out a warning and falling back on seeding with time if there is an error initializing.

Generally speaking, I think we should include a "/dev/urandom" and a "/dev/random" strategy in RandomSupport.SeedGenerator (which isn't necessarily a step you need to take for this PR). I think we should move away from the current default of using time in various Random procs, at least when "/dev/urandom" is available.